ground/src/main/res/layout/date_task_frag.xml

```diff
@@ -46,7 +45,6 @@
       android:maxLines="1"
       android:onClick="@{__ -> fragment.showDateDialog()}"
       android:text="@{viewModel.responseText}"
-      app:error="@{viewModel.error}"
```

Comment:
hope this is not breaking anything

Reply:
This was only done for date/time tasks and nothing else. But the error popup is shown for all tasks in a similar way.